src/main/java/com/google/devtools/build/lib/remote/RemoteModule.java

```diff
@@ -140,6 +149,22 @@ public void beforeCommand(CommandEnvironment env) throws AbruptExitException {
       return;
     }
 
+    RemoteOutputsStrategy remoteOutputsStrategy = remoteOptions.remoteOutputsStrategy;
+    if (remoteOutputsStrategy == RemoteOutputsStrategy.MINIMAL) {
```

Comment:
What do you think about extracting this logic into another private method, for example `validateOutputStrategy(...)`? This will improve readability of the `beforeCommand` method.

Reply:
done